pilota-build/test_data/thrift_workspace_with_split/output/article/src/gen.rs

```diff
@@ -0,0 +1,32 @@
+pub mod gen {
+    #![allow(warnings, clippy::all)]
+
+    pub mod article {
+        include!("article/enum_Status.rs");
+        include!("article/message_ArticleServiceGetArticleArgsRecv.rs");
+        include!("article/enum_ArticleServiceGetArticleResultSend.rs");
+        include!("article/message_GetArticleResponse.rs");
+        include!("article/message_ArticleServiceGetArticleArgsSend.rs");
+        include!("article/message_GetArticleRequest.rs");
+        include!("article/service_ArticleService.rs");
+        include!("article/enum_ArticleServiceGetArticleResultRecv.rs");
+        include!("article/message_Article.rs");
+
+        pub mod image {
+            include!("article/image/message_Image.rs");
+
+            pub mod cdn {
+                include!("article/image/cdn/message_CDN.rs");
+            }
+        }
```

Comment:
Great! Thanks very much for your quick fix!

My last feature request: could we make a more thorough split like the original design?

For example, we can put these all in `article/mod.rs` and include it, which contains the following content:

```rust
include!("article/enum_Status.rs");
include!("article/message_ArticleServiceGetArticleArgsRecv.rs");
include!("article/enum_ArticleServiceGetArticleResultSend.rs");
include!("article/message_GetArticleResponse.rs");
include!("article/message_ArticleServiceGetArticleArgsSend.rs");
include!("article/message_GetArticleRequest.rs");
include!("article/service_ArticleService.rs");
include!("article/enum_ArticleServiceGetArticleResultRecv.rs");
include!("article/message_Article.rs");

pub mod image {
    include!("article/image/mod.rs"); // recursively
}
```

I'm not sure if this will add extra cost to the IDEs, but I think this is a more rusty way and is much more extensible.

What do you think? @missingdays @Millione

Reply:
Agreed